docs/modules/release-notes/pages/all-releases/4_13_3S.adoc

```diff
@@ -0,0 +1,107 @@
+= Kobiton 4.13.3S release notes
+:navtitle: Kobiton 4.13.3S release notes
+
+_July 19, 2024_
+
+== New OS support
+
+Kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced iOS 17, they rolled out new connectivity protocols that impacted how iOS 17 devices communicated with solutions like Kobiton. These protocols required special network configurations for Kobiton Lightning Mode deployments that were not always possible for customers to implement.
+
+With iOS 17.4, Apple introduced the CoreDevice communication protocol that significantly enhances connections between iOS devices and the Kobiton platform.We implemented this innovative protocol with this release. CoreDevice eliminates the need for the special Lightning Mode networking changes introducedwith iOS 17 for iOS 17.4+ devices, simplifying your networking.
+
+Apple's CoreDevice cannot be used with iOS 17.0-17.3 devices. These devices must be hosted on a separate Mac mini in Standard mode.

```

Comment:
Change to: 
NOTE: iOS 17.0-17.3 devices continue to require special networking to support Lightning mode, so we recommend those devices are hosted on a separate Mac mini in Standard mode.

Reply:
updated